src/main/java/org/example/service/ConnectionService.java

```diff
@@ -0,0 +1,61 @@
+package org.example.service;
+
+import java.sql.*;
+
+
+public class ConnectionService {
+
+    private static final String DATABASE_NAME = "postgres";
+    private static final String DATABASE_USER = "postgres";
+    private static final String DATABASE_PASSWORD = "postgres";
+    private final Connection connection;
+
+    private static ConnectionService INSTANCE = null;
+
+    private static final String DRIVER_POSTGRESQL = "org.postgresql.Driver";
+
+    private static final String JDBC_URL = "jdbc:postgresql://developers-db:5432/";
+
+    public ConnectionService() {
+        this.connection = connectToDb();
+    }
+
+    public Connection getConnection() {
+        return connection;
+    }
+
+    public Connection connectToDb() {
+        Connection connection = null;
```

Comment:
`null` here is redundant

Reply:
fix

Reply:
Can we have a little discussion about that later. I will remind you that. And i decided to write down topics to discuss.

Reply:
Ok, let's go

Reply:
fixed